packages/create-plugin/scripts/recursiveFileCheck.mjs

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
@jackw remember to delete this before merging

Reply:
Hmm, couldn't we use it in our CI for some regression checks? Not as a blocking check, but something that could show us if the generated output changed - could be good for refactors. (The workflow could even be triggered by the `refactor` label, and then it could be blocking).

Reply:
It's a good idea but looking at it fresh from break none of this file is really necessary. We can use `diff` to recursively compare:

```
/usr/bin/diff -ry no-plop plop
{									{
diff -ry no-plop/myorg-plugin-app/.config/.cprc.json plop/myorg-plugin-app/.config/.cprc.json
  "version": "2.11.0-canary.602.81e7a0e.0"			|         "version": "2.10.1"
}									}
{									{
diff -ry no-plop/myorg-plugin-datasource/.config/.cprc.json plop/myorg-plugin-datasource/.config/.cprc.json
  "version": "2.11.0-canary.602.81e7a0e.0"			|         "version": "2.10.1"
}									}
{									{
diff -ry no-plop/myorg-plugin-panel/.config/.cprc.json plop/myorg-plugin-panel/.config/.cprc.json
  "version": "2.11.0-canary.602.81e7a0e.0"			|         "version": "2.10.1"
}									}
{									{
diff -ry no-plop/myorg-plugin-scenesapp/.config/.cprc.json plop/myorg-plugin-scenesapp/.config/.cprc.json
  "version": "2.11.0-canary.602.81e7a0e.0"			|         "version": "2.10.1"
}									}
{									{
diff -ry no-plop/myorg-pluginnobackend-datasource/.config/.cprc.json plop/myorg-pluginnobackend-datasource/.config/.cprc.json
  "version": "2.11.0-canary.602.81e7a0e.0"			|         "version": "2.10.1"
}									}
```

Something for the future IMHO as right now we cannot automate scaffolding until this PR is merged.